functions-methods.js

```diff
@@ -33,4 +57,21 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+function checkEmailValidity(emailadres) {
+    const at = emailadres.includes("@");
+    const comma = emailadres.includes(",");
+    const lastIndex = emailadres.charAt(emailadres.length - 1);
+
+    if (at === true && comma === false && lastIndex !== ".") {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Dit kun je nog iets inkorten 😉 
```suggestion
    return at && !comma && lastIndex !== ".";
```